packages/babel-plugin/src/testUtils.js

```diff
@@ -0,0 +1,14 @@
+import * as babel7 from '@babel/core'
```

Comment:
Maybe rename to just babel?

Reply:
I am working on tests to ensure babel6 works, so it will be needed